### PR TITLE
Deskto version

### PR DESCRIPTION
**Pull request by MMGGYY66**
Converting mobile version to desktop.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,274 +2,295 @@
...
-    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet" />
 </head>
 
 <body>
+  <!-- start main container -->
+  <main class="main-container">
+    <!-- start header section -->
```
Comment (tufoinnkuo10) [inline index.html:20]: - [ ] great work. Please ensure that as from screen size above 375, the mobile version should be responsive but well organized like the Figma. Please fix the text covering the background image at that screen size.
Screenshot (651)_LI

File: index.html [hunk 2]
```diff
@@ -2,274 +2,295 @@
...
+    <!-- start work section -->
+    <section class="recent-works" id="portfolio">
+      <div class="heading-recent-works">
+        <h2>My Recent works</h2>
+        <div class="border-recent-works">
+          <hr>
+        </div>
+      </div>
+      <div class="main">
+        <div class="main-image"></div>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please kindly add the multipost story image in both your desktop and mobile version, so your design matches the original Figma design. nice work

yours 
Screenshot (652)_LI
figma
Screenshot (653)

File: index.html [hunk 3]
```diff
@@ -2,274 +2,295 @@
...
 
-                        <a href="#"><img src="images/linkedin.png" alt="linkden-icon" /></a>
-                    </a></li>
-                <li><a href="">
+      <div class="programming-languages apply-flex">
+        <div class="language">
+          <h4 class="languages">Language</h4>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
```
Comment (tufoinnkuo10) [inline index.html:224]: - [ ] please kindly align the language frameworks and skills like that of figma.
yours
Screenshot (654)_LI
figma design
Screenshot (655)

File: index.html [hunk 4]
```diff
@@ -2,274 +2,295 @@
...
+        <input type="text" name="user_fullname" placeholder="Full name" maxlength="30" />
+        <div class="full-name apply-flex">
+          <input type="text" name="first_name" placeholder="First Name" maxlength="30" />
+          <input type="text" name="last_name" placeholder="Last Name" maxlength="30" />
+        </div>
+        <input type="email" name="user_email" placeholder="Email Adress" id="contact-me-email" required />
+        <textarea name="user_message" required maxlength="500">
+Enter text here</textarea>
+        <p id="email-error" class="error-message">
+          Your email should be written in lower case.
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] the error message is showing in your desktop and mobile version without the user trying to submit the form. please fix that

File: index.html [hunk 5]
```diff
@@ -2,274 +2,295 @@
...
+        </div>
+      </div>
+    </section>
 
-                        <a href="#"><img src="images/angellist.png" alt="angelist-icon" /></a>
-                    </a></li>
-                <li><a href="">
+    <section class="contact-section apply-flex" id="contact">
+      <h3>
+        I'm always interested in hearing about new projects, so if you'd like
```
Comment (tufoinnkuo10) [inline index.html:255]: - [ ] kindly use the correct font color for desktop version of text... `#262626`

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @MMGGYY66 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Media query well used
✅ Desktop design well implemented

Required Changes ♻️
Check the comments under the review
tufoin
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 6]
```diff
@@ -82,354 +75,334 @@ button:disabled {
...
+  font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 400;
-  font-size: 16px;
-  color: #f4f5f7;
-  padding: 12px 16px;
+  font-size: 22px;
+  line-height: 43px;
+  color: #fff;
 }
```
Comment (franclobo) [inline style.css]: - [ ] As the last code reviewer said 👉🏽 Kindly align the language frameworks and skills like that of Figma. Please 🙏🏽 make sure that both desktop and mobile versions look similar to the Figma template.

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MMGGYY66, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @MMGGYY66 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**